Add New Species: Xelthia

ZetaNull opened the pull request:
<!--
This is a semi-strict format, you can add/remove sections as needed but the order/format should be kept the same
Remove these comments before submitting
-->

# Description

<!--
Explain this PR in as much detail as applicable

Some example prompts to consider:
How might this affect the game? The codebase?
What might be some alternatives to this?
How/Who does this benefit/hurt [the game/codebase]?
-->
This is a PR to add a new species, the Xelthia. I find that most of the species in this game don't feel particularly alien, and I'm aware that they aren't as alien as some might want, but I feel like they're decently alien, personally. In addition to this, I've tried to set them up to fill niches that aren't particularly common with other species, such as having a heat resistance and cold weakness where other species tend to have cold resistances and heat weaknesses.

Here's their current list of (planned) traits, which is tentative and I'm open to adjusting.
### Negatives
- Cold weaknesses
- Lack of a gloves slot thanks to having tentacle arms
- Carnivores, will only eat things containing meat for the most part
### Positives
- Heat resistances
- Arm Regrowth, dropping their current arms
- Acid blood (Currently slippable)
- Caustic immunity (so they don't burn from falling in their own blood)
### ???
- Dropped arms are able to be eaten and/or cooked.

I've not worked with C# before now, so apologies if the code is a little bit scuffed or anything

---

# TODO

<!--
A list of everything you have to do before this PR is "complete"
You probably won't have to complete everything before merging but it's good to leave future references
-->

- [x] Implement Xelthia Species, complete with sprites and a language
- [x] Add species default names
- [x] Add displacement maps and such to make sure they look right
- [x] Implement primary YAML-based species functions, basically everything that isnt the limb regrowth function
- [x] Axe the initial psionics bit
- [x] Add the actual function for limb regrowth
- [x] Add proper lore
- [x] Any other possible considerations with adding Xelthia characteristics

---

<!--
This is default collapsed, readers click to expand it and see all your media
The PR media section can get very large at times, so this is a good way to keep it clean
The title is written using HTML tags
The title must be within the <summary> tags or you won't see it
-->

<details><summary><h1>Media</h1></summary>
<p>


https://github.com/user-attachments/assets/4fd3cb4f-4d5d-44dd-b5c0-62fd392e8920



![image](https://github.com/user-attachments/assets/3e5db9cc-a2b3-424e-a75c-cd410d57a0b2)

![image](https://github.com/user-attachments/assets/0a8e3f50-e20c-406e-8c1e-78cdd4eb1496)

![image](https://github.com/user-attachments/assets/bc860e97-6180-484c-afb1-e539decbb425)

![image](https://github.com/user-attachments/assets/331a620b-4c7f-4913-81db-a3fe235f58c1)

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl: Zeta_Null
- add: Added Xelthia, a custom alien species
- add: Added an insulated jacket, to account for Xelthia not having access to a glove slot.
- add: Xelthia tentacle item that drops when arms are regrown. This is able to be cooked.

ZetaNull commented:
Xelthia are stumbling into MULTIPLE bugs with displacement maps while I work on them apparently lol (they're minor issues but :V)

ZetaNull commented:
I. I think this is it actually???? Assuming I've not missed anything and my implementation doesn't cause any errors, it's. Actually done?????

could always add more customization options and stuff, but as a starting point

ZetaNull commented:
WHY IS THE YAML LINTER CHOKING???

EctoplasmIsGood commented:
ZETA!!! GIVE XELTHIA INCREASED REGEN, AND MY LIFE, IS YOURS.

ZetaNull commented:
okay nobody merge this for right now actually I need to add another xelthia jacket and play with adding some small amount of regen with arm regrowth

ZetaNull commented:
okay yeah small regen when you regrow arms is in but a medical jacket might have to wait I can't be bothered to look into it right now and if this passes I'm fine with it being merged lol

ZetaNull commented:
is there a good way to rerun tests if they decide to random fail

VMSolidus commented:
Yes, I can just manually tell the tests to run again.

ZetaNull commented:
alright cool I'll just ask about it next time it does that

MLGTASTICa commented:
Their laugh is really high pitched , any chance it could get toned down to lower frequencies ? right now it feels like one of those dog-deterrent lamps (if you've ever heard one)

Plyushsune commented:
![image-7](https://github.com/user-attachments/assets/55c853c1-13a9-4336-90d1-5b9e7dbc2ff4)

Timfa2112 commented on Content.Server/_EE/Xelthia/XelthiaSystem.cs:
```diff
@@ -0,0 +1,116 @@
+        var end = _gameTiming.CurTime + component.UseDelay;
+        _actionsSystem.SetCooldown(component.XelthiaRegenerateAction!.Value, start, end); // Cooldown
+
+        //IEntityManager entityManager = uid;
+        IEntityManager entityManager = base.EntityManager; // There's no way this is good code. I don't know what im doing though, so. I dunno.
+        var bodySystem = entityManager.System<BodySystem>();
+        var transformSystem = entityManager.System<SharedTransformSystem>();
+
+        if (!entityManager.TryGetComponent(uid, out BodyComponent? body)
+            || !entityManager.TryGetComponent(uid, out TransformComponent? xform))
+            return;
+
+        var root = bodySystem.GetRootPartOrNull(uid, body);
+        if (root is null)
+            return;
+
+        var parts = bodySystem.GetBodyChildrenOfType(uid, BodyPartType.Arm, body); // Deletes Arms and hands
+        foreach (var part in parts)
+        {
+
+            foreach (var child in bodySystem.GetBodyPartChildren(part.Id, part.Component))
+                entityManager.QueueDeleteEntity(child.Id);
+
+            transformSystem.AttachToGridOrMap(part.Id);
+            entityManager.QueueDeleteEntity(part.Id);
+            entityManager.SpawnAtPosition("FoodMeatXelthiaTentacle", xform.Coordinates); // Should drop arms equal to the number attached
```
```suggestion
        var bodySystem = _entityManager.System<BodySystem>();
        var transformSystem = _entityManager.System<SharedTransformSystem>();

        if (!_entityManager.TryGetComponent(uid, out BodyComponent? body)
            || !_entityManager.TryGetComponent(uid, out TransformComponent? xform))
            return;

        var root = bodySystem.GetRootPartOrNull(uid, body);
        if (root is null)
            return;

        var parts = bodySystem.GetBodyChildrenOfType(uid, BodyPartType.Arm, body); // Deletes Arms and hands
        foreach (var part in parts)
        {

            foreach (var child in bodySystem.GetBodyPartChildren(part.Id, part.Component))
                _entityManager.QueueDeleteEntity(child.Id);

            transformSystem.AttachToGridOrMap(part.Id);
            _entityManager.QueueDeleteEntity(part.Id);
            _entityManager.SpawnAtPosition("FoodMeatXelthiaTentacle", xform.Coordinates); // Should drop arms equal to the number attached
```
Test this before you apply it!

Timfa2112 commented on Content.Server/_EE/Xelthia/XelthiaSystem.cs:
```diff
@@ -0,0 +1,116 @@
+            entityManager.SpawnAtPosition("FoodMeatXelthiaTentacle", xform.Coordinates); // Should drop arms equal to the number attached
+        }
+        // Right Side
+        var newLimb = entityManager.SpawnAtPosition("RightArmXelthia", xform.Coordinates); // Copy-Pasted Code for arm spawning
+        if (entityManager.TryGetComponent(newLimb, out BodyPartComponent? limbComp))
+            bodySystem.AttachPart(root.Value.Entity, "right arm", newLimb, root.Value.BodyPart, limbComp);
+
+        var newerLimb = entityManager.SpawnAtPosition("RightHandXelthia", xform.Coordinates); // Spawns the hand
+        Enum.TryParse<BodyPartType>("Hand", out var partType);
+        bodySystem.TryCreatePartSlotAndAttach(newLimb, "right hand", newerLimb, partType);
+
+        // Left Side
+        newLimb = entityManager.SpawnAtPosition("LeftArmXelthia", xform.Coordinates); // Copy-Pasted Code for arm spawning
+        if (entityManager.TryGetComponent(newLimb, out BodyPartComponent? limbComp2))
+            bodySystem.AttachPart(root.Value.Entity, "left arm", newLimb, root.Value.BodyPart, limbComp2);
+
+        newerLimb = entityManager.SpawnAtPosition("LeftHandXelthia", xform.Coordinates); // Spawns the hand
+        bodySystem.TryCreatePartSlotAndAttach(newLimb, "left hand", newerLimb, partType);
+
+        entityManager.EntitySysManager.GetEntitySystem<SharedAudioSystem>()
```
```suggestion
        var newLimb = _entityManager.SpawnAtPosition("RightArmXelthia", xform.Coordinates); // Copy-Pasted Code for arm spawning
        if (_entityManager.TryGetComponent(newLimb, out BodyPartComponent? limbComp))
            bodySystem.AttachPart(root.Value.Entity, "right arm", newLimb, root.Value.BodyPart, limbComp);

        var newerLimb = _entityManager.SpawnAtPosition("RightHandXelthia", xform.Coordinates); // Spawns the hand
        Enum.TryParse<BodyPartType>("Hand", out var partType);
        bodySystem.TryCreatePartSlotAndAttach(newLimb, "right hand", newerLimb, partType);

        // Left Side
        newLimb = _entityManager.SpawnAtPosition("LeftArmXelthia", xform.Coordinates); // Copy-Pasted Code for arm spawning
        if (_entityManager.TryGetComponent(newLimb, out BodyPartComponent? limbComp2))
            bodySystem.AttachPart(root.Value.Entity, "left arm", newLimb, root.Value.BodyPart, limbComp2);

        newerLimb = _entityManager.SpawnAtPosition("LeftHandXelthia", xform.Coordinates); // Spawns the hand
        bodySystem.TryCreatePartSlotAndAttach(newLimb, "left hand", newerLimb, partType);

        _entityManager.EntitySysManager.GetEntitySystem<SharedAudioSystem>()
```

Timfa2112 commented on Content.Shared/_EE/Xelthia/XelthiaComponent.cs:
```diff
@@ -0,0 +1,22 @@
+[RegisterComponent, NetworkedComponent]
+public sealed partial class XelthiaComponent : Component
+{
+    /// <summary>
+    /// These should store the colors for the Xelthia's spikes when regrown? I'm unsure how necessary this strictly is
+    /// but I imagine I'll need to have this for the color to persist.
+    /// </summary>
```
Verify that this code is working correctly

Timfa2112 commented on Content.Shared/_EE/Xelthia/XelthiaSystem.cs:
```diff
@@ -0,0 +1 @@
+//Moved to Server because of issues
```
Delete this file if it's no longer necessary.

Timfa2112 commented on Resources/Locale/en-US/_EE/language/languages.ftl:
```diff
@@ -17,3 +17,7 @@ language-YaSsa-description =
+
+language-Qiilour-name = Qiilour
+language-Qiilour-description =
+    The native primary language of the Xelthia, just about all of them speak it in some capacity. It's a blend of several older languages that have since been lost to time.
```
```suggestion
    The native and principal language of the Xelthia, Qiilour is spoken by nearly all members of their species to some degree. It is a refined fusion of several ancient tongues—such as Telenari, Voskarn, and Old Xelthec—each now lost to history.
```
optional

Timfa2112 commented on Resources/Locale/en-US/_EE/reagents/xelthia.ftl:
```diff
@@ -0,0 +1,5 @@
+reagent-desc-acid-blood = Like blood, but it burns you.
+
+reagent-name-armjuice = Xelthia Arm Juice
+reagent-desc-armjuice = This saps your hunger and thirst. Where are you getting this from.
```
```suggestion
reagent-desc-armjuice = An organic fluid extracted from Xelthian physiology. Rather than sustaining the body, it dramatically increases hunger and thirst.
```

Timfa2112 commented on Resources/Prototypes/Traits/Psionics/feats.yml:
```diff
@@ -3,11 +3,13 @@
   category: TraitsPsionicsFeats
   points: -5
   requirements:
-    - !type:CharacterTraitRequirement
```
Changes to psionic traits, which are being removed, are not related to this species. They should be reverted to avoid problems in the near future.

Timfa2112 commented on Resources/Prototypes/_EE/Actions/xelthia.yml:
```diff
@@ -0,0 +1,12 @@
+- type: entity
+  id: XelthiaRegenerateAction
+  name: Regrow Arms
+  description: Regrow your lost arms. Tentacles? Tentacle arms.
```
```suggestion
  description: Regrow any limbs you may have lost.
```

Timfa2112 commented on Resources/Prototypes/_EE/Body/Prototypes/xelthia.yml:
```diff
@@ -0,0 +1,50 @@
+﻿- type: body
+  name: "xeltha"
```
```suggestion
  name: "xelthia"
```

Timfa2112 commented on Resources/Prototypes/_EE/Entities/Clothing/OuterClothing/jackets.yml:
```diff
@@ -0,0 +1,14 @@
+    sprite: _EE/Clothing/InsulatedJacket/insulated_jacket.rsi
+  - type: Insulated
+  - type: FingerprintMask
+# PLEASE FIX DISPLACEMENT MAPS APPLYING TO SPECIES VARIANTS OF CLOTHES THERES NO WAY ITS INTENTIONAL hgsjkd (i have a version that's normal for if/when that behavior is changed. Seems like this would be an oversight)
```
Seconded--this type of comment should not be left here. If a change to displacement maps is needed, make a Github issue, where people will actually see it.

Timfa2112 commented on Resources/Prototypes/_EE/Entities/Mobs/Customization/Markings/xelthia.yml:
```diff
@@ -0,0 +1,128 @@
+
+- type: marking
+  id: XelthiaBackspikes
+  bodyPart: Chest, RArm, LArm, Head
```
Verify that this is working as it should be

Timfa2112 commented on Resources/Prototypes/_EE/Entities/Mobs/Species/xelthia.yml:
```diff
@@ -0,0 +1,264 @@
+  name: Urist McXelthia
+  abstract: true
+  components:
+  - type: Hunger
+  - type: Icon # It will not have an icon in the adminspawn menu without this. Body parts seem fine for whatever reason.
+    sprite: _EE/Mobs/Species/Xelthia/parts.rsi
+    state: full
+  - type: Body
+    prototype: Xelthia
+  - type: Thirst
```
Is this needed? Verify that their dietary needs are mechanically enforced

Timfa2112 commented on Resources/Prototypes/_EE/Entities/Mobs/Species/xelthia.yml:
```diff
@@ -0,0 +1,264 @@
+#        parameterUV: displacementUV
+  - type: Inventory
+    templateId: xelthia
+    speciesId: xelthia
```
```suggestion
    speciesId: Xelthia
```

Timfa2112 commented on Resources/Prototypes/_EE/Entities/Mobs/Species/xelthia.yml:
```diff
@@ -0,0 +1,264 @@
+            layerKey: dummy
+            parameterTexture: displacementMap
+            parameterUV: displacementUV
+      suitstorage:
+        layer:
+          sprite: _EE/Mobs/Species/Xelthia/displacement.rsi
+          state: jumpsuit
+          copyToShaderParameters:
+            # Value required, provide a dummy. Gets overridden when applied.
+            layerKey: dummy
+            parameterTexture: displacementMap
+            parameterUV: displacementUV
+      eyes:
+        layer:
+          sprite: _EE/Mobs/Species/Xelthia/displacement.rsi
+          state: jumpsuit
+          copyToShaderParameters:
+            # Value required, provide a dummy. Gets overridden when applied.
+            layerKey: dummy
+            parameterTexture: displacementMap
+            parameterUV: displacementUV
+      belt:
+        layer:
+          sprite: _EE/Mobs/Species/Xelthia/displacement.rsi
+          state: jumpsuit
+          copyToShaderParameters:
+            # Value required, provide a dummy. Gets overridden when applied.
+            layerKey: dummy
+            parameterTexture: displacementMap
+            parameterUV: displacementUV
+      back:
+        layer:
+          sprite: _EE/Mobs/Species/Xelthia/displacement.rsi
+          state: backpack
+          copyToShaderParameters:
+            # Value required, provide a dummy. Gets overridden when applied.
+            layerKey: dummy
+            parameterTexture: displacementMap
+            parameterUV: displacementUV
+
```
Please verify that these displacement maps are set up correctly

Timfa2112 commented on Resources/Prototypes/_EE/InventoryTemplates/xelthia_inventory_template.yml:
```diff
@@ -0,0 +1,124 @@
+      uiWindowPos: 1,2
+      strippingWindowPos: 1,1
+      displayName: Mask
+    - name: eyes
+      slotTexture: glasses
+      slotFlags: EYES
+      stripTime: 3
+      uiWindowPos: 0,3
+      strippingWindowPos: 0,0
+      displayName: Eyes
```
Can you test these items in-game and verify that they work?

Timfa2112 commented on Resources/Prototypes/_EE/Reagents/biological.yml:
```diff
@@ -0,0 +1,67 @@
+      - !type:Emote
+        emote: Scream
+        probability: 0.3
+#      conditions: # TODO: Figure out how to make this work properly, it doesnt seem like it wants to do anything when uncommented lol
+#      - !type:OrganType
+#        type: Xelthia
+#        shouldHave: false
```
Can you try to fix this?

Timfa2112 commented on Resources/Prototypes/_EE/Reagents/biological.yml:
```diff
@@ -0,0 +1,77 @@
+        factor: -4
+      - !type:SatiateHunger
+        factor: -4
+      - !type:HealthChange # this originally had tricord's effects but then it was giving weird heal numbers in testing?? so I've reduced the scope. feel free to add more later I just want to be done working on them for now lol
```
Please verify what's happening with Tricordrazine--if this change is intentional, then the comment is not needed.

Timfa2112 commented on Resources/Prototypes/_EE/Species/xelthia.yml:
```diff
@@ -0,0 +1,194 @@
+    Underwear: MobHumanoidAnyMarking
+    Undershirt: MobHumanoidAnyMarking
+    Chest: MobXelthiaTorso
+    HeadTop: MobHumanoidAnyMarking
+    HeadSide: MobHumanoidAnyMarking
+    Hair: MobHumanoidAnyMarking
+    Tail: MobHumanoidAnyMarking
+    Eyes: MobXelthiaEyes
+    LArm: MobXelthiaLArm
```
Please verify that these default markings are applied correctly

Timfa2112 commented on Resources/Textures/_EE/Contractors/Passport.rsi/meta.json:
```diff
@@ -54,6 +54,12 @@
         },
         {
             "name": "passport_goldendeep_open"
+        },
+		{
+            "name": "passport_qiimaeru_closed"
+        },
```
```suggestion
        },
	{
            "name": "passport_qiimaeru_closed"
        },
```

Timfa2112 commented on Resources/ServerInfo/_EE/Guidebook/Mobs/Xelthia.xml:
```diff
@@ -0,0 +1,27 @@
+  
+  ## Gameplay Information
+  
+  [color=#00cc00][bold]Positives:[/bold][/color]
+  
+  - Thanks to having tentacles, are able to drop and regrow their arms as needed; dropped arms are able to be cooked and eaten if desired.
+  - Have a higher-than-average heat tolerance, and take [color=#ffa500]less Heat damage[/color] overall.
+  - Have [color=#bbff00]Acidic blood[/color], slippable.
+  - Immune to most sources of [color=#bbff00]Caustic damage[/color].
+  
+  [color=red][bold]Negatives:[/bold][/color]
+  
+  - Carnivores; can only eat foods containing meat.
+  - Thanks to having tentacles for arms, they [color=red]can't wear gloves[/color]; if you want [color=yellow]electrical insulation[/color], you'll have to track down an [color=yellow]insulated jacket[/color]!
+  - Have worse-than average cold tolerance, and take more [color=#1e90ff]Cold damage[/color] overall.
+  
+  ## Background Information
+  To sum them up briefly, Xelthia are a species that have only had their first contact with humans around 25 years ago. They tend to be foodies, and generally aren't huge fans of wasting things, going as far with it as to often eat their own dead— though cooked of course. Xelthia generally tend to fall into two camps; those who are extremely driven to do things and accomplish their own goals, and ...
+  
+  [color=#878787](More in-depth lore is to be written at a later date, though some details include that they're originally from a swamp planet and the specific year of their first contact is likely around 2424. This existing summary will stay regardless for those who aren't as interested or are in a rush.)[/color]
```
```suggestion
[color=#00cc00][bold]Positives:[/bold][/color]

- Possessing tentacular limbs, Xelthia can detach and later regrow their arms at will. Detached arms can be cooked and consumed, if desired.
- Exhibit a higher-than-average resistance to heat, taking [color=#ffa500]reduced Heat damage[/color] overall.
- Their [color=#bbff00]acidic blood[/color] is not only dangerous, but also creates slippable surfaces.
- Naturally immune to most forms of [color=#bbff00]Caustic damage[/color].

[color=red][bold]Negatives:[/bold][/color]

- Obligate carnivores; they can only consume foods that contain meat.
- Due to their tentacle-like arms, [color=red]gloves cannot be worn[/color]. If [color=yellow]electrical insulation[/color] is needed, an [color=yellow]insulated jacket[/color] will be required instead.
- Poor resistance to cold, resulting in increased vulnerability to [color=#1e90ff]Cold damage[/color].

## Background Information
The Xelthia are a relatively new presence on the interstellar stage, having made first contact with humanity roughly 25 years ago. Known for their culinary curiosity, they despise waste—going so far as to consume their own dead when necessary (though always thoroughly cooked). Culturally, they tend to fall into two broad groups: those with intense personal ambition and drive, and those who prefer to simply blend into society, often taking on more routine roles such as janitorial work.

[color=#878787](Additional lore will be added in the future. Notably, the Xelthia originate from a swamp-dominated world, and their first contact with humanity is estimated to have occurred around the year 2424. This summary will remain available for those seeking a quick overview.)[/color]

```

Timfa2112 commented on Resources/Prototypes/_EE/Species/xelthia.yml:
```diff
@@ -0,0 +1,194 @@
+      defaultMarkings: [ XelthiaBackspikes ]
+    Underwear:
+      points: 1
+      required: false
+    Undershirt:
+      points: 1
+      required: false
```
Make sure this list is working properly please

Timfa2112 reviewed (changes requested):
Some things that need to be changed, a lot of things that need to be checked, and some alternate descriptions that read a bit better.

ZetaNull commented:
Some comments on some of the requests, setting up a PR with anything I've not mentioned here though right now

> Test this before you apply it!

Will be applying and testing shortly

> Verify that this code is working correctly

I forgot to remove that when I changed how the backspikes work, removing it in my next PR

> Changes to psionic traits, which are being removed, are not related to this species. They should be reverted to avoid problems in the near future.

I forgot to fully revert that when removing psionics from earlier versions of Xelthia, sorry about that lol

> description: Regrow any limbs you may have lost.

I'm going to replace the word limbs here with arms if that's alright? I don't want to imply legs regrowing when they don't.

> Seconded--this type of comment should not be left here. If a change to displacement maps is needed, make a Github issue, where people will actually see it.

yeah fair, i did half have it there as a reminder that if it gets fixed that that needs editing but I could just. Make a comment that's better at actually doing that, like this

![image](https://github.com/user-attachments/assets/5308bda3-4523-46ff-b675-a4918b2d2588)


> Verify that this is working as it should be

It works, I have it set up like that because I saw other things set up like this and it was otherwise rendering under the arms and head (which it doesn't now)

![image](https://github.com/user-attachments/assets/2c84543a-5669-4815-b501-88711f2edb7e)

> Is this needed? Verify that their dietary needs are mechanically enforced

Their diet is handled with their stomach organ, I don't think a diet component like coderabbit suggested exists? Though I could be wrong, i've just not like. Seen that anywhere.

> Please verify that these displacement maps are set up correctly

They are, I just reused the jumpsuit displacement maps because they ended up being fairly generically applicable without having to make a bunch of new ones

> Can you test these items in-game and verify that they work?

Eyes slot works fine, I had the Xelthia sprite set up so that glasses actually do cover their lower eyes.

![image](https://github.com/user-attachments/assets/aa12e602-c2d0-4047-9ff1-8376a626e018)

> Can you try to fix this?

I could double check but I tried what coderabbit suggested for the acid blood not burning Xelthia when drank after it didnt work with the original attempt, the only reason I reverted back to that was so it wouldnt be commented out three times (though maybe it would have been better to do that to like. Show that had been attempted.)

> Please verify what's happening with Tricordrazine--if this change is intentional, then the comment is not needed.

Comment can be deleted

> Please verify that these default markings are applied correctly

the only markings that Xelthia actually seem to get ingame that aren't exclusive to them are earrings which seem to apply fine

![image](https://github.com/user-attachments/assets/1558150d-caf0-4f3d-94ee-cf45d6617e80)

> [changes to xelthia guidebook entry]

This is the only bit of text that i'm a little bit iffy on changing? Not a particularly big deal, though i prefer the simpler feel of the original version, at least regarding the gameplay features section and and summary (though I could probably still refine it a bit)

I'd probably want to be more in line with that kind of feel for the proper lore section once that's added, these sections in particular just feel like they might warrant being a bit more casual imo

> Make sure this list is working properly please

I don't see what this is referring to unless it's the spike marking's list or whatever, which. I've previously mentioned has been tested and can confirm works fine.

I don't know why coderabbit is pointing at the _EE/Species/xelthia.yml file here???
